From 542d446210ac6d74b7d54b04de76f04c8acfbbdf Mon Sep 17 00:00:00 2001
From: Suhaib <suhaib92khan@gmail.com>
Date: Mon, 8 Jun 2020 17:02:55 -0400
Subject: [PATCH] Add unit test for getRatio function

---
 client_test.py | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/client_test.py b/client_test.py
index 5175a7f..8f2f0d4 100644
--- a/client_test.py
+++ b/client_test.py
@@ -1,5 +1,5 @@
 import unittest
-from client3 import getDataPoint
+from client3 import getDataPoint, getRatio
 
 class ClientTest(unittest.TestCase):
   def test_getDataPoint_calculatePrice(self):
@@ -23,7 +23,13 @@ class ClientTest(unittest.TestCase):
 
   """ ------------ Add more unit tests ------------ """
 
+  def test_getRatio_calculateRatioDivingZero(self):
+    prices = [119.2, 0, 121.68, 117.87]
+    self.assertEqual(getRatio(prices[0], prices[1]), None)
 
+  def test_getRatio_calculateRatio(self):
+    prices = [119.2, 0, 121.68, 117.87]
+    self.assertEqual(getRatio(prices[2], prices[3]), (prices[2] / prices[3]))
 
 if __name__ == '__main__':
     unittest.main()
-- 
2.23.0

